plugin_engine.py

```diff
@@ -96,8 +97,18 @@ def load_plugins(app_context: PluginAppContext):
             try:
                 plugin = importlib.import_module(name)
                 getattr(plugin, "__plugin_entrypoint__")(app_context)
+                active_plugins[name.split(".")[1]] = {
```

Comment:
All modules have `__name__` attribute
```suggestion
                active_plugins[plugin.__name__] = {
```